example/src/example/core.cljs

```diff
@@ -103,7 +103,35 @@
         :pixi.texture/source     "img/bunny.png"}
        :pixi.object/filters
        [{:pixi.filter/fragment outline-shader
-         :pixi.filter/uniforms {:dimensions {:type "2f" :value [400.0 300.0]}}}]}}})}})
+         :pixi.filter/uniforms {:dimensions {:type "2f" :value [400.0 300.0]}}}]}}}
+    :e
+    {:impi/key :text
+     :pixi.object/type :pixi.object.type/text
+     :pixi.object/position [100 290]
+     :pixi.text/text "Hello World!"
+     :pixi.text/style 
+     {:pixi.text.style/align "center"
+      :pixi.text.style/break-words true
+      :pixi.text.style/drop-shadow true
+      :pixi.text.style/drop-shadow-angle (/ Math.PI 6)
+      :pixi.text.style/drop-shadow-blur 0
+      :pixi.text.style/drop-shadow-color 0x222222
+      :pixi.text.style/drop-shadow-distance 5
+      :pixi.text.style/fill 0xFFFF00
+      :pixi.text.style/font-family "Arial"
+      :pixi.text.style/font-size 24
+      ;:pixi.text.style/font-style "bold"
```

Comment:
Probably want to remove this line or change it to something like "italic", right?